src/modules/formBuilder/components/formTree/FormTreeLabel.tsx

```diff
@@ -91,6 +112,47 @@ const FormTreeLabel: React.FC<FormTreeLabelProps> = ({
         height: '48px',
       }}
     >
+      {!!itemDependents && (
+        <ConfirmationDialog
```

Comment:
Might be nice to pull this out into a separate file, since it's pretty far from "Form Tree Label". This can be simplified to `<DeleteItemConfirmationDialog dependentItems={dependentItems} />`